contracts/Lens/VenusLens.sol

```diff
@@ -584,14 +584,34 @@ contract VenusLens is ExponentialNoError {
     }
 
     /**
-     * @notice Calculate the total XVS tokens pending or accrued by a user account
+     * @dev Struct for Pending Rewards for per market
+     */
+    struct PendingReward {
+        address vTokenAddress;
+        uint256 amount;
+    }
+
+    /**
+     * @dev Struct for Reward of a single reward token.
+     */
+    struct Reward {
+        address rewardTokenAddress;
+        uint256 totalRewards;
+        PendingReward[] pendingRewards;
+    }
+
+    /**
+     * @notice Calculate the total XVS tokens pending and accrued by a user account
      * @param holder Account to query pending XVS
      * @param comptroller Address of the comptroller
-     * @return Total number of accrued XVS that can be claimed
+     * @return Reward object contraining the totalRewards and pending rewards for each market
      */
-    function pendingVenus(address holder, ComptrollerInterface comptroller) external view returns (uint) {
+    function pendingVenus(address holder, ComptrollerInterface comptroller) external view returns (Reward memory) {
```

Comment:
Maybe we should rename the function as well then?

```suggestion
    function pendingRewards(address holder, ComptrollerInterface comptroller) external view returns (Reward memory) {
```

Reply:
Would it make sense to create a new function, keeping the legacy one as it was?

Reply:
Not really sure if the old function can be used for something, because it gives the same information. Maybe we can leave the same name? @therealemjy  can you give your opinion?
Do we need to have the same interface for both VenusLens and PoolLens?

Reply:
The old one won't be used anymore once this is deployed, so no need to keep it for legacy purposes.
Regarding the interface, if I understand well both share basically the same interface, except the function from `VenusLens` returns a single `Reward` instead of an array of `Reward`. Then I think what you did there is good.

Reply:
Funciton name has been changed